### DISCUSSION TITLE
Go to symbol in workspace is slow

### DISCUSSION THREAD

**Discussion by dev-joss** [microsoft/vscode-cpptools#10426] [General]
Finding that Go to symbol in workspace (CTRL + T) is slow, making it impossible to use.

Seems like cpptools.exe is taking up the time, writing some precompiled header..?

Is there anything I can do to speed this up? Already excluded everything in windows defender.

Similar functionality in Visual Studio is nice and snappy.

**Comment by sean-mcmanus** [comment]
We have 2 issues tracking this https://github.com/microsoft/vscode-cpptools/issues/7908 and https://github.com/microsoft/vscode-cpptools/issues/7914 .
